pkg/client/client.go

```diff
@@ -331,6 +331,26 @@ func (client *Client) GetUser(ctx context.Context, userName string) (jira.User,
 	}, err
 }
 
+func (client *Client) GetUserByKey(ctx context.Context, userKey string) (*UsersAPIData, error) {
+	var userAPIData UsersAPIData
+	req, err := getRequest(ctx, client, baseUserPath, Query{
```

Comment:
We have to let zscaler know that we are now making a GET request for this path, because they have to update the method + URL path we can access whenever we change this.